meinberlin/apps/plans/models.py

```diff
@@ -96,20 +119,32 @@ class Plan(UserGeneratedContentModel):
     )
     topics = TopicField(
         verbose_name=_('Topics'),
-        help_text=_('Add topics to your project.'))
+        help_text=_('Assign your plan to 1 or 2 '
+                    'topics. In the project '
+                    'overview projects can be '
+                    'filtered according to topics.'))
     status = models.SmallIntegerField(
         choices=STATUS_CHOICES,
-        verbose_name=_('Status')
+        verbose_name=_('Status'),
+        help_text=_('In the project overview projects '
+                    'can be filtered by status.')
     )
     participation = models.SmallIntegerField(
         choices=PARTICIPATION_CHOICES,
-        verbose_name=_('Participation')
+        verbose_name=_('Participation'),
+        help_text=_('In the project overview '
+                    'projects can be filtered '
+                    'according to participation '
+                    'status. ')
```

Comment:
Here is a blank too much at the end. I should have had a look before the translations. Now, it can stay like that (also because we don't show the English in mB).